release/local.sh

```diff
@@ -20,21 +20,16 @@ SCRIPT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
 stage "Content Provider - Local Folder"
 
 pkg_list="${WORK_DIR}/${PKG_LIST_TMP}local"
+pkg_list_files="${WORK_DIR}/${PKG_LIST_TMP}local-files"
```

Comment:
There is meaning behind the ${PKG_LIST_TPM}<variant> on workdir. Each fine with that pattern is concatenated by content.sh to form the final pkg list, so this may cause all local files to be added into that list twice.